testproject/Assets/Tests/Runtime/Support/SpawnRpcDespawn.cs

```diff
@@ -0,0 +1,148 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using MLAPI;
+using MLAPI.Configuration;
+using MLAPI.Logging;
+using MLAPI.Messaging;
+using MLAPI.Spawning;
+using NUnit.Framework;
+using UnityEngine;
+using Object = System.Object;
+
+namespace TestProject.RuntimeTests.Support
+{
+    public class SpawnRpcDespawnInstanceHandler : INetworkPrefabInstanceHandler
```

Comment:
Not a big fan of multiple classes in the same file, I'd put that in it's own file.